src/Views/Format.elm

```diff
@@ -54,6 +55,14 @@ formatFoodSelectedImpact { trigram, unit, decimals } =
         >> formatRichFloat decimals unit
 
 
+formatFoodSelectedImpactPerKg : Impact.Definition -> Mass -> Impacts -> Html msg
+formatFoodSelectedImpactPerKg { trigram, unit, decimals } mass =
+    Impact.getImpact trigram
+        >> Unit.impactToFloat
+        >> (\impact -> impact / Mass.inKilograms mass)
+        >> formatRichFloat decimals unit
```

Comment:
So this is just and only an Html view thing? Don't we ever expect to have to expose this unitin API results? 

Shouldn't we rather have a dedicated function in the `Unit` module?

Reply:
There's at the moment nothing in `Unit` related to `Impacts` or `Impact.Definition` or `Html`, so I'm not sure what you're suggesting? This helper is the pendant of `formatFoodSelectedImpact` defined just above.

Reply:
https://github.com/MTES-MCT/ecobalyse/blob/master/src/Data/Unit.elm#L393-L440 ?

Reply:
This isn't the same `Impact.Definition` we're talking about, is it? I'm confused

Reply:
I think we should use `formatFoodSelectedImpact` and do the "per kg" conversion before using it. That with or without introducing some helper in `Unit`, like we did for `impactAggregateScore` for example.

Reply:
bah scratch it